web/templates/coping_strategy/form.html.eex

```diff
@@ -1,15 +1,21 @@
-<p class="b">Note down an action you can take to stay well.
+<p class="b">Note down actions you can take if you notice a change in your feelings, thoughts or behaviour.
 <p>
-  Something you can do, or avoid doing when you notice a change in your
-  feelings, thoughts or behaviour.
+What could you do, or avoid doing?<br>
+Is there something you know has helped in the past?
 </p>
-<p>For example, practice relaxation techniques, leave the room if things are
-  too tense, take a break or tell someone you trust.</p>
+<p>For example,<br>
+If I start to feel anxious I can:<br>
+- practice relaxation techniques<br>
```

Comment:
For semantic I think we could use the ```li``` tags with some css to replace the point with a dash.